Automattic/wp-calypso
# Migration trial: Update migration trial step design (version 2)

Closes #80499

## Proposed Changes

* Update Migration step design

## Testing Instructions

* Go to `/setup/import-focused/migrationTrial?siteSlug={SITE_SLUG}&from={JN_SITE}&option=everything`
* Check if there is a new version of the migration trial step

<img width="841" alt="Screenshot 2023-08-14 at 00 07 23" src="https://github.com/Automattic/wp-calypso/assets/1241413/a4d7bc44-1413-4876-b21f-d397316094d4">



## Pre-merge Checklist
